src/GetPlasterTemplate.ps1

```diff
@@ -10,6 +10,22 @@ function Get-PlasterTemplate {
         [string]
         $Path,
 
+        [Parameter(Position=1,
+        ParameterSetName="Path",
```

Comment:
Thanks!  This will be a very nice enhancement! Would you mind indenting all the parameter fields under the first line like the rest of the Parameter attributes?  Should we add the `[ValidateNotNullOrEmpty()]` attribute?

Reply:
Good pickup. I have fixed the indentation and added the validate.